docs/circuit_cutting/explanation/index.rst

```diff
@@ -60,10 +60,10 @@ As a basic and explicit example, let us consider the decomposition of a cut :cla
 
 As shown in [`2 <https://arxiv.org/abs/1909.07534>`__], a quantum circuit which implements an :class:`~qiskit.circuit.library.RZZGate` can be simulated by performing six subexperiments where the :class:`~qiskit.circuit.library.RZZGate` in the original circuit has been replaced with only local (single-qubit) operations [the :math:`\mathcal{F}_i`\ 's in Eq. :eq:`eq:qpd`].  The result is then reconstructed by combining the subexperiment results with certain coefficients [the :math:`a_i`\ 's in Eq. :eq:`eq:qpd`], which can be either positive or negative.  Given the :math:`\theta` parameter of the :class:`~qiskit.circuit.library.RZZGate`, the six subexperiments are as follows:
 
-1. With coefficient :math:`a_1 = \cos^2 \theta`, do nothing (:math:`I \otimes I`, where :math:`I` is the identity operation on a single qubit).
-2. With coefficient :math:`a_2 = \sin^2 \theta`, perform a :class:`~qiskit.circuit.library.ZGate` on each qubit (:math:`Z \otimes Z`).
-3. With coefficient :math:`a_3 = \sin(2\theta)/2`, perform a projective measurement in the Z basis on the first qubit and an :class:`~qiskit.circuit.library.SGate` gate on the second qubit (denote this as :math:`M_z \otimes S`).  If the result of the measurement is 1, flip the sign of that outcome's contribution during reconstruction.
-4. With coefficient :math:`a_4 = -\sin(2\theta)/2`, perform a projective measurement in the Z basis on the first qubit and an :class:`~qiskit.circuit.library.SdgGate` gate on the second qubit (denote this as :math:`M_z \otimes S^\dagger`).  If the result of the measurement is 1, flip the sign of that outcome's contribution during reconstruction.
+1. With coefficient :math:`a_1 = \cos^2 (\theta/2)`, do nothing (:math:`I \otimes I`, where :math:`I` is the identity operation on a single qubit).
+2. With coefficient :math:`a_2 = \sin^2 (\theta/2)`, perform a :class:`~qiskit.circuit.library.ZGate` on each qubit (:math:`Z \otimes Z`).
+3. With coefficient :math:`a_3 = -\sin\theta/2`, perform a projective measurement in the Z basis on the first qubit and an :class:`~qiskit.circuit.library.SGate` gate on the second qubit (denote this as :math:`M_z \otimes S`).  If the result of the measurement is 1, flip the sign of that outcome's contribution during reconstruction.
+4. With coefficient :math:`a_4 = \sin \theta/2`, perform a projective measurement in the Z basis on the first qubit and an :class:`~qiskit.circuit.library.SdgGate` gate on the second qubit (denote this as :math:`M_z \otimes S^\dagger`).  If the result of the measurement is 1, flip the sign of that outcome's contribution during reconstruction.
```

Comment:
I believe restoring the parentheses will remove ambiguity.

![no-parentheses](https://github.com/user-attachments/assets/55f5794a-e6e4-4e1f-aa1e-724dd9dd5c8e)


```suggestion
3. With coefficient :math:`a_3 = -\sin(\theta)/2`, perform a projective measurement in the Z basis on the first qubit and an :class:`~qiskit.circuit.library.SGate` gate on the second qubit (denote this as :math:`M_z \otimes S`).  If the result of the measurement is 1, flip the sign of that outcome's contribution during reconstruction.
4. With coefficient :math:`a_4 = \sin(\theta)/2`, perform a projective measurement in the Z basis on the first qubit and an :class:`~qiskit.circuit.library.SdgGate` gate on the second qubit (denote this as :math:`M_z \otimes S^\dagger`).  If the result of the measurement is 1, flip the sign of that outcome's contribution during reconstruction.
```

Reply:
I did think that I added parenthesis in the most recent commit ([89a76bf](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/657/commits/89a76bf56279e3feaefdf01dfdb71a9c2bd7bb05))

Reply:
Sorry, I actually missed these ones. My bad.